_utils/configs/darwin.ini

```diff
@@ -0,0 +1,40 @@
+# https://archive.mozilla.org/pub/firefox/releases/latest/README.txt
+
```

Comment:
per our conversation, we'll tackle OSX and linux later.  these paths may more-or-less work, but they'll need verifying as well as a bit of OS-specific stuff like dmg-mounting for OSX.  cygwin, however is working fine